src/preload.js

```diff
@@ -12,14 +12,18 @@ const notificationPath = `${app.getPath("userData")}/notification.jpg`;
 let currentSong = "";
 let player;
 let currentPlayStatus = statuses.paused;
+let barvalue;
+let updatecurrent = false;
+let oldcurrent;
+let currentURL = undefined;
 
 const elements = {
   play: '*[data-test="play"]',
   pause: '*[data-test="pause"]',
   next: '*[data-test="next"]',
   previous: 'button[data-test="previous"]',
   title: '*[data-test^="footer-track-title"]',
-  artists: '*[data-test^="grid-item-detail-text-title-artist"]',
+  artists: '*[class^="elemental__text elemental__text css-oxcos"]',
```

Comment:
Did you make this change on purpose? Been using the data-test for a few days now and haven't gotten any issues.

Reply:
For some reason my tidal keeps messing around with the artist name when I use the data-test I will send a pic of what it does when I get on my computer.

Reply:
Oh.. it fetches the WRONG artist. Gotcha.